.github/workflows/ci.yml

```diff
@@ -66,9 +31,12 @@ jobs:
           sudo xcode-select -switch /Applications/Xcode_12.app
       - name: build
         run: |
-          export PATH="/usr/local/opt/lua@5.3/lib:/usr/local/opt/lua@5.3/include:/usr/local/opt/lua@5.3/bin:$PATH"
+          export PATH="/usr/local/opt/lua@5.3/lib:/usr/local/opt/lua@5.3/include:/usr/local/opt/lua@5.3/bin:/usr/local/opt/flex/bin:$PATH"
           export PKG_CONFIG_PATH="/usr/local/opt/lua@5.3/lib/pkgconfig"
-          ./bootstrap.sh -e -t && cd build  && cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_LUA_SCRIPTING=1 -DENABLE_SQL=ON -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_RULE_MESSAGES=OFF -DSTRICT_GSL_CHECKS=AUDIT -DFAIL_ON_WARNINGS=ON .. && cmake --build . --parallel 4
+          export LDFLAGS="-L/usr/local/opt/flex/lib"
+          export CPPFLAGS="-I/usr/local/opt/flex/include"
+          # CPPFLAGS are not recognized by cmake, so we have to force them to CFLAGS and CXXFLAGS to have flex 2.6 working
```

Comment:
CPP here: **C** **P**re**p**rocessor
CPPFLAGS is normally the place where you want to put your include paths and definitions. Not to be confused with CXXFLAGS, which are C++ compiler flags the same way CFLAGS are C compiler flags.
In most cases, including ours, cpp is called by the compiler driver, but it's possible to call it separately, similarly to as and ld.